src/utils/network_info.ts

```diff
@@ -408,6 +408,17 @@ const networks = {
     label: "Kava - KAVA",
     name: "Kava",
   },
+  "kava-testnet": {
+    address: "kavavaloper1vlpsrmdyuywvaqrv7rx6xga224sqfwz3yjnlkh",
+    delegate: "https://wallet.keplr.app/chains/kava", // FIXME test chain not found in keplr https://testnet.keplr.app/
```

Comment:
I think could remove this line, this `delegate` property is a legacy feature and not important in new platform

Reply:
You are right.
I forgot to push my changes 🙈.